backend/src/main/java/pt/ua/deti/ies/SmartHomes/backend/RabbitMQ/Listener.java

```diff
@@ -15,19 +17,29 @@
 @Slf4j
 @Service
 public class Listener {
+
     @Autowired
     private WriteApi writeApi;
 
+    @Autowired
+    private WebSocketSender webSocketSender;
+
     @RabbitListener(queues = {QUEUE_NAME})
     public void consumeMessage(final Message message) {
         try {
             log.info("Received message: [{}]", message);
             writeDataToInfluxDB(message);
+            writeToSocket(message);
         } catch (Exception e) {
             log.error("Error while writing to InfluxDB: {}", e.getMessage());
         }
     }
 
+    private void writeToSocket(Message message) {
+        webSocketSender.sendMessageToTopic("electricity", message);
```

Comment:
Thinking in a future perspective, it would be great to also set the id on the topic. Something like "0/electricity", so that multi-home support is easier to implement.
```suggestion
        webSocketSender.sendMessageToTopic(message.getId() + "electricity", message);
```